server/src/main/java/io/crate/replication/logical/plan/DropSubscriptionPlan.java

```diff
@@ -51,8 +55,37 @@ public void executeOrFail(DependencyCarrier dependencies,
                               RowConsumer consumer,
                               Row params,
                               SubQueryResults subQueryResults) throws Exception {
+
+        // This is POC - takes only first table and utilizes existing requests for close/open.
+        // Test passes because it has only one table so findFirst doesn't hurt.
+        // If it works, close/open requests must be adapted to get list of DocTableInfo
+
+        // Step 1 - Close subscribed tables and consequently stop tracking and remove retention lease.
+        DocTableInfo tableInfo  = (DocTableInfo) InformationSchemaIterables.tablesStream(dependencies.schemas())
+            .filter(t -> {
+                if (t instanceof DocTableInfo dt) {
+                    return analyzedDropSubscription.name().equals(REPLICATION_SUBSCRIPTION_NAME.get(dt.parameters()));
+                }
+                return false;
+            })
+            .findFirst().get();
```

Comment:
actually we take all list and adapt requests, was just going to make POC locally